
We'd quite like fatal errors to create core files that allow analysis.

diff -wpruN '--exclude=*.orig' a~/src/libs/zbxnix/sighandler.c a/src/libs/zbxnix/sighandler.c
--- a~/src/libs/zbxnix/sighandler.c	1970-01-01 00:00:00
+++ a/src/libs/zbxnix/sighandler.c	1970-01-01 00:00:00
@@ -189,12 +189,6 @@ void	zbx_set_common_signal_handlers(void
 	sigaction(SIGTERM, &phan, NULL);
 	sigaction(SIGUSR2, &phan, NULL);
 
-	phan.sa_sigaction = fatal_signal_handler;
-	sigaction(SIGILL, &phan, NULL);
-	sigaction(SIGFPE, &phan, NULL);
-	sigaction(SIGSEGV, &phan, NULL);
-	sigaction(SIGBUS, &phan, NULL);
-
 	phan.sa_sigaction = alarm_signal_handler;
 	sigaction(SIGALRM, &phan, NULL);
 }
